src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php

```diff
@@ -327,6 +327,14 @@ protected function createForm(string $type, $data = null, array $options = []):
         return $this->container->get('form.factory')->create($type, $data, $options);
     }
 
+    /**
+     * Creates and returns a Form view from the type of the form.
+     */
+    protected function createFormView(string $type, $data = null, array $options = []): FormView
+    {
+        return $this->container->get('form.factory')->create($type, $data, $options)->createView();
```

Comment:
It seems to me a special case. Most of the time forms must handle the request. Here we would lose that opportunity, I think it's not so common to generalize.

Reply:
I had exactly the same feeling when looking at the changes. Does it really happen that often that you need to display a form without having having to process its submission somehow?

Reply:
Most of the cases are multistep forms or forms that are way to simple like only a submit button.

You have a controller with `firstStepAction` that is viewing a `FirstStepForm`. The `FirstStepForm` gets you redicted to the `secondStepAction` on which you actually handle the request using the `firstStepForm`.. Could it be done differently.. maybe.. but that not the point of discussion i think...

> I think it's not so common to generalize.

I do respect that.. if you feel that this doesn't fit here.. then it's fine...

Reply:
Actually, I just do not know that. Just because I didn't encounter this use case doesn't mean others don't have it. I think best is to gather some more feedback.

Reply:
I had the same initial thought: "is it really that common"?
As implementing this in a trait in separate library is pretty straightforward, I am not sure whether this is worth adding it to the base controller (but you can say that about a lot of the methods there..)